blueprints/ember-cli-moment-shim/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  normalizeEntityName: function() {
+    // this prevents an error when the entityName is
+    // not specified (since that doesn't actually matter
+    // to us
+  },
+
+  afterInstall: function() {
+    return this.addBowerPackageToProject('ember-cli-moment-shim');
```

Comment:
Should it be both?

I'm not entirely sure how `ember-cli-moment-shim` operates.

Would it make sense to add `moment` as a bower dependency, then `require('moment-shim')()` in the appropriate place?

What would the appropriate place be? `index.js`?